src/style-auro.scss

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// Import Auro CSS Properties as Sass file
+@import "./node_modules/@alaskaairux/orion-design-tokens/dist/tokens/SassCustomProperties";
+
+// Import Auro Sass variables
+@import "./node_modules/@alaskaairux/orion-design-tokens/dist/tokens/SCSSVariables";
+
+// general WCSS imports
+@import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/breakpoints";
+@import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/core";
+@import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/animation";
+
+@import "./node_modules/@alaskaairux/orion-design-tokens/dist/tokens/SCSSVariableMap";
+
+$auro-inset-spacing-options: lg;
+$auro-inset-directions:'--squish';
+@import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/utilityMixins/insetUtility";
+
+$color-base-black-opacity-15: rgba(255, 255, 255, 0.15);
+
+// handle focus-visible state
+:host(.focus-visible) {
+  .auro-button {
+    box-shadow: inset 0 0 0 1px var(--auro-color-border-focus-on-dark), inset 0 0 0 3px var(--auro-color-background-lightest);
+
+    // auro-buttonOndark
+    &Ondark {
+      background-color: var(--auro-color-ui-default-on-dark);
```

Comment:
Double-check your values here. The style is being applied, but the UI is not correct.

Reply:
What specifically is broken on the UI?